acs-service-setup/dumps/uns.yaml

```diff
@@ -0,0 +1,45 @@
+# This dump creates a service account for the Sparkplug Ingester service
+# and grants it the necessary permissions to read and write to the UNS.
+# It also adds the ReadEntireUNS permission to the Warehouse role so
+# that the UNS historian can use it and has permission to read the UNS.
+---
+service: !u UUIDs.Service.ConfigDB
+version: 1
+overwrite: true
+objects:
+  !u ACS.Class.Permission:
+    - !u ACS.Perm.MQTT.WriteToEntireUNS
+    - !u ACS.Perm.MQTT.ReadEntireUNS
+  !u ACS.Class.ServiceAccount:
+    - !u ACS.ServiceAccount.SparkplugIngester
+    - !u ACS.ServiceAccount.HistorianUNS
+configs:
+  !u UUIDs.App.Info:
+    !u ACS.Perm.MQTT.WriteToEntireUNS: { name: "MQTT: Write to entire UNS" }
+    !u ACS.Perm.MQTT.ReadEntireUNS: { name: "MQTT: Read entire UNS" }
+    !u ACS.ServiceAccount.SparkplugIngester: { name: "Ingester (Sparkplug)" }
+    !u ACS.ServiceAccount.HistorianUNS: { name: "Historian (UNS)" }
+  !u UUIDs.App.MQTTPermissionTemplate:
+    !u ACS.Perm.MQTT.WriteToEntireUNS: "UNS/v1/#: w"
+    !u ACS.Perm.MQTT.ReadEntireUNS: "UNS/v1/#: rs"
+---
+service: !u UUIDs.Service.Authentication
+version: 1
+principals:
+  - uuid: !u ACS.ServiceAccount.SparkplugIngester
+    kerberos: sv1sparkplugingester@{{realm}}
```

Comment:
@amrc-benmorrow how do we reference the realm here? I thought it might get templated but it doesn't seem to (image below is from deployment of `v3.3.0-dev.2`.

![image](https://github.com/user-attachments/assets/f09a9f73-b8b3-4196-8d63-bf72dc79cb07)

Reply:
You don't, you let krbkeys create these entries. These service account UUIDs should not be fixed in any case, they should be per-installation.
* Create an Auth group _Sparkplug Ingesters_. This does have a fixed UUID.
* Grant that group appropriate permissions. If all it needs is _Global Debuggers_ then you don't need a new group but I think it will need UNS publish permission as well? You should be able to make the _Sparkplug Ingesters_ group a member of the _Global Debuggers_ group, which grants read-and-rebirth.
* Request account creation on the `sv1sparkplugingester` KerberosKey. The [central Monitor](https://github.com/AMRC-FactoryPlus/amrc-connectivity-stack/blob/45407a96f5ffb65fe6e0441064fb9c37809438d8/deploy/templates/auth/principals/service-clients.yaml#L116) is the only account to use this pattern so far but it's how I'd like to do things going forward.